<!DOCTYPE HTML>
<html>
<head>
  <title>Observation-genetics-cg-prf-1a - FHIR v0.5.0</title>

  
  <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
  <meta name="author" content="http://hl7.org/fhir"/>

  <link rel="stylesheet" href="fhir.css"/>
  <link rel="Prev" href="http://hl7.org/fhir/observation-genetics-cg-prf-1a"/>

    <!-- Bootstrap core CSS -->
  <link rel="stylesheet" href="./dist/css/bootstrap.css"/>
  <link rel="stylesheet" href="./assets/css/bootstrap-fhir.css"/>

    <!-- Project extras -->
  <link rel="stylesheet" href="./assets/css/project.css"/>
  <link rel="stylesheet" href="./assets/css/pygments-manni.css"/>
	<link rel="stylesheet" href="jquery-ui.css"/>

    <!-- HTML5 shim and Respond.js IE8 support of HTML5 elements and media queries -->
    <!-- [if lt IE 9]>
  <script src="./assets/js/html5shiv.js"></script>
  <script src="./assets/js/respond.min.js"></script>
  <![endif] -->

    <!-- Favicons -->
  <link sizes="144x144" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-144-precomposed.png"/>
  <link sizes="114x114" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-114-precomposed.png"/>
  <link sizes="72x72" rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-72-precomposed.png"/>
  <link rel="apple-touch-icon-precomposed" href="./assets/ico/apple-touch-icon-57-precomposed.png"/>
  <link rel="shortcut icon" href="./assets/ico/favicon.png"/>

</head>
<body>
	<div id="segment-header" class="segment">  <!-- segment-header -->	
		<div class="container">  <!-- container -->
			<a id="logo" href="http://hl7.org/fhir"><img src="./assets/images/fhir-logo-www.png" alt="logo fhir"/> </a>
			<div id="hl7-status">
				<b>Sequence Resource Proposal Build</b>
			</div>       	
			
			<div id="hl7-nav">
				 <a id="hl7-logo" href="http://www.hl7.org">
					<img src="./assets/images/hl7-logo.png" width="42" alt="visit the hl7 website" height="50"/>
				</a> 
			</div>       	
			  <!-- hl7-search -->
		</div>
		<div class="container">  <!-- container -->
	</div></div>  <!-- /segment-header -->	


	<div id="segment-navbar" class="segment">  <!-- segment-navbar -->
		<div id="stripe"> </div>
		<div class="container">  <!-- container -->
   <!-- HEADER CONTENT -->
 	      		
 	      		      		
			<nav class="navbar navbar-inverse">
				<div class="container">
					<button data-target=".navbar-inverse-collapse" data-toggle="collapse" type="button" class="navbar-toggle">
						<span class="icon-bar"> </span>
						<span class="icon-bar"> </span>
						<span class="icon-bar"> </span>
					</button>
					<a href="index.html" class="navbar-brand hidden">FHIR</a>
					<div class="nav-collapse collapse navbar-inverse-collapse">
						<ul class="nav navbar-nav">
              <li><a href="index.html">Home</a></li>
              <li><a href="documentation.html">Documentation</a></li>
              <li><a href="implementation.html">Implementation</a></li>
              <li><a href="resourcelist.html">Resources</a></li>
              <li><a href="clinical.html">Clinical</a></li>
              <li><a href="administration.html">Administrative</a></li>
              <li><a href="infrastructure.html">Infrastructure</a></li>
              <li><a href="financial.html">Financial</a></li>

						</ul>
					</div>  <!-- /.nav-collapse -->
				</div>  <!-- /.container -->
			</nav>  <!-- /.navbar -->
      		
				
  <!-- /HEADER CONTENT -->				
		</div>  <!-- /container -->
	</div>  <!-- /segment-navbar -->
	

	<div id="segment-breadcrumb" class="segment">  <!-- segment-breadcrumb -->
		<div class="container">  <!-- container -->	
			<ul class="breadcrumb">
        <li><a href="index.html">Home</a></li>
        <li><a href="clinical.html">Clinical</a></li>        <li><a href="observation.html">Observation</a></li>        <li><a href="observation-genetics-cg-prf-1a.html">Profile</a></li>        <li><b>Standard Profile for Genetics</b></li>          <!-- observation-genetics-cg-prf-1a.html / profile:Observation/observation-genetics-cg-prf-1a / Standard Profile for Genetics -->

			</ul>	
		</div>  <!-- /container -->
	</div>  <!-- /segment-breadcrumb -->
	

	<div id="segment-content" class="segment">  <!-- segment-content -->
	<div class="container">  <!-- container -->
            <div class="row">
            	<div class="inner-wrapper">
  <!-- CONTENT CONTENT -->


<div class="col-9">

<h2><span class="sectioncount">4.20.13<a name="4.20.13"> </a></span> Standard Profile for Genetics <a href="observation-genetics-cg-prf-1a.html#4.20.13" title="link to here" class="self-link"><img src="target.png"/></a></h2>



<h3><span class="sectioncount">4.20.13.1<a name="4.20.13.1"> </a></span> Overview <a href="observation-genetics-cg-prf-1a.html#4.20.13.1" title="link to here" class="self-link"><img src="target.png"/></a></h3>
<p>
Observation-genetics-profile (i.e. Standard Profile for Genetics) extends <a href="observation.html">Observation</a> resource to enable reporting of structured genetic test results. In addition, the genetics profile contextualizes well established standards from the field of clinical genetics into the standards of healthcare (e.g. HGNC - HUGO Gene Nomenclature Committee's international standard for gene names, symbols, and identifiers).
</p>
<p>
<h4><span class="sectioncount">4.20.13.1.1<a name="4.20.13.1.1"> </a></span> Genetic Standards and Resources include: <a href="observation-genetics-cg-prf-1a.html#4.20.13.1.1" title="link to here" class="self-link"><img src="target.png"/></a></h4>
			<ul> 
				<li>Variant Databases: dbSNP, ClinVar, and COSMIC </li>
				<li>Reference Sequences: RefSeq and ENSEMBL</li>
				<li> Gene Symbols and Identifiers: HGNC - Human Genee Nomenclature Committee</li>
				<li>Variant Nomenclature: HGVS nomenclature from the Human Genome Variantion Society</li>
				<li>Variant Feature Annotation: Sequence Ontology (SO) and LOINC</li>
			</ul>
</p>
<h3><span class="sectioncount">4.20.13.2<a name="4.20.13.2"> </a></span> Scope and Usage <a href="observation-genetics-cg-prf-1a.html#4.20.13.2" title="link to here" class="self-link"><img src="target.png"/></a></h3>
<p>
The Standard Profile for Genetics supports reporting of a DNA variant at the genomic, cDNA, and protein change level. In addition, a condition context may be provided, as AssessedCondition. For large genomic tests, a condition may be used as an input into the analytic pipeline to aid in the identification of clinically relevant variants related to the test order. It is strongly encouraged to provide all available information in this profile for any reported variants, because receiving systems (e.g. discovery research, outcomes analysis, and public health reporting) may use this information to normalize variants over time or across sources.  However, these data should not be used to dynamically correct/change variant representations for clinical use outside of the laboratory, due to insufficient information.

</p>



<a name="content"> </a>
<h3><span class="sectioncount">4.20.13.3<a name="4.20.13.3"> </a></span> Content <a href="observation-genetics-cg-prf-1a.html#4.20.13.3" title="link to here" class="self-link"><img src="target.png"/></a></h3>

<table class="lines"><tr><td colspan="2"><b>Profiles</b>: </td></tr><tr><td><a href="observation-genetics-cg-prf-1a-genetics.html">Genetics</a></td><td>Describes how the observation resource is used to report structured genetic test results</td></tr><tr><td colspan="2"><b>Extensions</b>: </td></tr><tr><td><a name="extension-geneticsGenomeBuild"/><a href="extension-geneticsgenomebuild.html">geneticsGenomeBuild</a></td><td><b>GenomeBuild</b> : <p>The Genome Build used for reference, following GRCh build versions e.g. 'GRCh 37'.  Version number must be included if a versioned release of a primary build was used.</p>
</td></tr><tr><td><a name="extension-geneticsChromosome"/><a href="extension-geneticschromosome.html">geneticsChromosome</a></td><td><b>Chr</b> : <p>The chromosome containing the genetic finding - values should be 1-23, X, Y, mitochondrial, viral, bacteria.</p>
</td></tr><tr><td><a name="extension-geneticsGenomicReferenceSequenceId"/><a href="extension-geneticsgenomicreferencesequenceid.html">geneticsGenomicReferenceSequenceId</a></td><td><b>GenomicRefSeqID</b> : <p>The genomic reference sequence identifier is included, to support testing using alternative reference sequences.</p>
</td></tr><tr><td><a name="extension-geneticsGenomicStart"/><a href="extension-geneticsgenomicstart.html">geneticsGenomicStart</a></td><td><b>GenomicStart</b> : <p>Nucleotide location for start of genomic finding on the positive (+) genomic strand.</p>
</td></tr><tr><td><a name="extension-geneticsGenomicStop"/><a href="extension-geneticsgenomicstop.html">geneticsGenomicStop</a></td><td><b>GenomicStop</b> : <p>Nucleotide location for end of genomic finding on the positive (+) genomic strand.</p>
</td></tr><tr><td><a name="extension-geneticsReferenceAllele"/><a href="extension-geneticsreferenceallele.html">geneticsReferenceAllele</a></td><td><b>RefAllele</b> : <p>Nucleotide(s) from genomic start to genomic stop on the positive (+) strand of the genomic reference sequence. Use '-' for insertions.</p>
</td></tr><tr><td><a name="extension-geneticsObservedAllele"/><a href="extension-geneticsobservedallele.html">geneticsObservedAllele</a></td><td><b>ObsAllele</b> : <p>Oserved nucleotides from genomic start to genomic stop on the positive (+) genomic strand. Use '-' for delections.</p>
</td></tr><tr><td><a name="extension-geneticsGeneId"/><a href="extension-geneticsgeneid.html">geneticsGeneId</a></td><td><b>HGNCgene</b> : <p>HGNC gene symbol and identifier from HUGO Gene Nomenclature Committee at: http://www.genenames.org/.</p>
</td></tr><tr><td><a name="extension-geneticsTranscriptReferenceSequenseId"/><a href="extension-geneticstranscriptreferencesequenseid.html">geneticsTranscriptReferenceSequenseId</a></td><td><b>TransReferenceSeqID</b> : <p>Reference identifier for cDNA transcript, with version, from NCBI's RefSeq or ENSEMBL.</p>
</td></tr><tr><td><a name="extension-geneticsDNASequenceVariation"/><a href="extension-geneticsdnasequencevariation.html">geneticsDNASequenceVariation</a></td><td><b>HGVSdnaVariant</b> : <p>cDNA variant following HGVS nomenclature on the given TranscriptReferenceSequenceId.</p>
</td></tr><tr><td><a name="extension-geneticsVariationId"/><a href="extension-geneticsvariationid.html">geneticsVariationId</a></td><td><b>VariantId</b> : <p>Identifier for variant.  If a germline variant, ClinVar or dbSNP identifier should be used. If ClinVar is used, the identifier should be prefixed with 'ClinVar:' (e.g. ClinVar:17661).  dbSNP ids are start with 'rs' (rs28897672) and COSMIC id's start with 'COSM' (e.g. COSM12979). Note, dbSNP id's are not unique to a variant but a location, so nucleotide change and germline/somatic classification are needed to resolve.  If a somatic variant, COSMIC identifier should be used, unless in ClinVar then either maybe used.</p>
</td></tr><tr><td><a name="extension-geneticsDNARegionName"/><a href="extension-geneticsdnaregionname.html">geneticsDNARegionName</a></td><td><b>RegionName</b> : <p>Details of exonic location of variant (e.g. Exon 1).</p>
</td></tr><tr><td><a name="extension-geneticsDNASequenceVariationType"/><a href="extension-geneticsdnasequencevariationtype.html">geneticsDNASequenceVariationType</a></td><td><b>DNAvariantType</b> : <p>Codified type for associated DNA Sequence Variation.  DNA Sequence Variations use the HGVS notation which implies the DNA Sequence Variation Type, but the concurrent use of this code will allow a standard and explicit type for technical and display convenience. LOINC Answer List values 48019-4 or Sequence Ontology vaues.</p>
</td></tr><tr><td><a name="extension-geneticsProteinReferenceSequenceId"/><a href="extension-geneticsproteinreferencesequenceid.html">geneticsProteinReferenceSequenceId</a></td><td><b>ProteinReferenceSeq</b> : <p>Reference identifier for protein transcript, with version, from NCBI's RefSeq or ENSEMBL.</p>
</td></tr><tr><td><a name="extension-geneticsAminoAcidChange"/><a href="extension-geneticsaminoacidchange.html">geneticsAminoAcidChange</a></td><td><b>HGVSproteinChange</b> : <p>Protein variant following HGVS nomenclature on the given ProteinReferenceSequenceId.</p>
</td></tr><tr><td><a name="extension-geneticsAminoAcidChangeType"/><a href="extension-geneticsaminoacidchangetype.html">geneticsAminoAcidChangeType</a></td><td><b>ProteinChangeType</b> : <p>Type of variation expressed using Sequence Ontology or LOINC answer list 48006-1.</p>
</td></tr><tr><td><a name="extension-geneticsAlleleName"/><a href="extension-geneticsallelename.html">geneticsAlleleName</a></td><td><b>AlleleName</b> : <p>Common name for variant or gene allele.</p>
</td></tr><tr><td><a name="extension-geneticsGenomicSourceClass"/><a href="extension-geneticsgenomicsourceclass.html">geneticsGenomicSourceClass</a></td><td><b>GenomicSource</b> : <p>The genomic class of the variant: Germline for inherited genome, somatic for cancer genome (e.g. DNA from tumor cells), and prenatal for fetal genome. Associated with LOINC answer list: 48002-0.</p>
</td></tr><tr><td><a name="extension-geneticsAllelicState"/><a href="extension-geneticsallelicstate.html">geneticsAllelicState</a></td><td><b>AllelicState</b> : <p>Level of occurence of the DNA variation in relation to the genomic context.</p>
</td></tr><tr><td><a name="extension-geneticsSpecies"/><a href="extension-geneticsspecies.html">geneticsSpecies</a></td><td><b>Species</b> : <p>Supports testing of human, virus, or bacteria.</p>
</td></tr><tr><td><a name="extension-geneticsCIGAR"/><a href="extension-geneticscigar.html">geneticsCIGAR</a></td><td><b>CIGAR</b> : <p>A sequence of of base lengths and the associated operation, used to indicate which bases align (either a match/mismatch) with the reference, are deleted from the reference, and are insertions that are not in the reference. string of observed nucleotides.  Observed nucleotides matching the reference are in capital letters. Observed nucleotides not matching the reference are in lower case letters. Use '-' a dash for deleted/missing nucleotides in the observed sequence. Allowable characters are A,T,C,G, a,t,c,g and -.</p>
</td></tr><tr><td><a name="extension-geneticsAssessedCondition"/><a href="extension-geneticsassessedcondition.html">geneticsAssessedCondition</a></td><td><b>AssessedCondition</b> : <p>Used to denote the condition context for genetic testing, which may influence reported variants and interpretation for large genomic testing panels e.g. lung cancer or familial breast cancer.</p>
</td></tr><tr><td colspan="2"><b>Examples</b>: </td></tr><tr><td><a href="obs-genetics-example1-somatic.html">Genetics Observation Example1 Somatic</a></td><td><p>Genetics Observation of Somatic Variant in Lung Cancer</p>
</td></tr><tr><td><a href="obs-genetics-example2-germline.html">Genetics Observation Example2 Germlin</a></td><td><p>Genetics Observation of Germline BRCA1 Variant</p>
</td></tr></table><a name="search"> </a>
<h3><span class="sectioncount">4.20.13.4<a name="4.20.13.4"> </a></span> Search Parameters <a href="observation-genetics-cg-prf-1a.html#4.20.13.4" title="link to here" class="self-link"><img src="target.png"/></a></h3>
<p>Search parameters defined by this package. See <a href="search.html">Searching</a> for more information about searching in REST, messaging, and services.</p>
<table class="list">
<tr><td><b>Name</b></td><td><b>Type</b></td><td><b>Description</b></td><td><b>Paths</b></td><td><b>Source</b></td></tr>
<tr><td>amino-acid-change</td><td><a href="search.html#string">string</a></td><td>HGVS Protein Change</td><td/><td><a href="observation-genetics-cg-prf-1a-Observation-amino-acid-change.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-amino-acid-change.json.html">JSON</a></td></tr>
<tr><td>assessed-condition</td><td><a href="search.html#string">string</a></td><td>Condition assessed by genetic test</td><td/><td><a href="observation-genetics-cg-prf-1a-Observation-assessed-condition.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-assessed-condition.json.html">JSON</a></td></tr>
<tr><td>chromosome-genomicstart</td><td><a href="search.html#string">string</a></td><td>Chromosome and Genomic Starting Position</td><td/><td><a href="observation-genetics-cg-prf-1a-Observation-chromosome-genomicstart.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-chromosome-genomicstart.json.html">JSON</a></td></tr>
<tr><td>condition-gene</td><td><a href="search.html#string">string</a></td><td>HGNC gene symbol and assessed condition context</td><td/><td><a href="observation-genetics-cg-prf-1a-Observation-condition-gene.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-condition-gene.json.html">JSON</a></td></tr>
<tr><td>condition-gene-dnavariant</td><td><a href="search.html#string">string</a></td><td>HGNC gene symbol  HGVS DNA variant, and assessed condition context</td><td/><td><a href="observation-genetics-cg-prf-1a-Observation-condition-gene-dnavariant.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-condition-gene-dnavariant.json.html">JSON</a></td></tr>
<tr><td>dna-variant</td><td><a href="search.html#string">string</a></td><td>HGVS DNA variant</td><td/><td><a href="observation-genetics-cg-prf-1a-Observation-dna-variant.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-dna-variant.json.html">JSON</a></td></tr>
<tr><td>gene-amino-acid-change</td><td><a href="search.html#string">string</a></td><td>HGNC gene symbol and HGVS Protein change</td><td/><td><a href="observation-genetics-cg-prf-1a-Observation-gene-amino-acid-change.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-gene-amino-acid-change.json.html">JSON</a></td></tr>
<tr><td>gene-dnavariant</td><td><a href="search.html#string">string</a></td><td>HGNC gene symbol and HGVS DNA Variant</td><td/><td><a href="observation-genetics-cg-prf-1a-Observation-gene-dnavariant.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-gene-dnavariant.json.html">JSON</a></td></tr>
<tr><td>gene-identifier</td><td><a href="search.html#string">string</a></td><td>HGNC gene symbol and identifier</td><td/><td><a href="observation-genetics-cg-prf-1a-Observation-gene-identifier.xml.html">XML</a> / <a href="observation-genetics-cg-prf-1a-Observation-gene-identifier.json.html">JSON</a></td></tr>
</table>




</div>


				</div>  <!-- /inner-wrapper -->
            </div>  <!-- /row -->
        </div>  <!-- /container -->
        
    </div>  <!-- /segment-content -->


	<div id="segment-footer" class="segment">  <!-- segment-footer -->
		<div class="container">  <!-- container -->
			<div class="inner-wrapper">
				<p>
        © HL7.org 2011+. FHIR DSTU (v0.5.0-5997) generated on Tue, Jul 28, 2015 17:34-0400.   <!-- <a href="qa.html">QA Report</a> -->   <!-- achive note -->
        <br/>
        <span style="color: #FFFF77">
        Links: <a style="color: #81BEF7" href="timelines.html">What's a DSTU?</a> | 
               <a style="color: #81BEF7" href="history.html">Version History</a> | 
               <a style="color: #81BEF7" href="toc.html">Table of Contents</a> | 
               <a style="color: #81BEF7" href="http://services.w3.org/htmldiff?doc1=http%3A%2F%2Fhl7.org%2Fimplement%2Fstandards%2Ffhir%2Fobservation-genetics-cg-prf-1a&amp;doc2=http%3A%2F%2Fhl7.org%2Ffhir%2FDSTU1%2Fobservation-genetics-cg-prf-1a">Compare to DSTU</a> | 
               <a rel="license" style="color: #81BEF7" href="license.html"><img src="cc0.png" alt="CC0" style="border-style: none;"/></a> | 
               <a style="color: #81BEF7" href="http://gforge.hl7.org/gf/project/fhir/tracker/?action=TrackerItemAdd&amp;tracker_id=677" target="_blank">Propose a change</a>   
        </span>
        </p>
			</div>  <!-- /inner-wrapper -->
		</div>  <!-- /container -->
	</div>  <!-- /segment-footer -->
  <!-- disqus thread -->
  <!-- disqus -->
  <!-- end disqus -->        
	
	<div id="segment-post-footer" class="segment hidden">  <!-- segment-post-footer -->
		<div class="container">  <!-- container -->
		</div>  <!-- /container -->
	</div>  <!-- /segment-post-footer -->
    
      <!-- JS and analytics only. -->
      <!-- Bootstrap core JavaScript
================================================== -->
  <!-- Placed at the end of the document so the pages load faster -->
<script src="./assets/js/jquery.js"> </script>     <!-- note keep space here, otherwise it will be transformed to empty tag -> fails -->
<script src="./dist/js/bootstrap.min.js"> </script>
<script src="./assets/js/respond.min.js"> </script>

<script src="./assets/js/fhir.js"> </script>

  <!-- Analytics Below
================================================== -->

    
</body>
</html>